docker/docker-chaos/storage-schemas.conf

```diff
@@ -54,3 +54,4 @@
 pattern = .*
 retentions = 1s:1d
 # reorderBuffer = 20
+# allowUpdate = true
```

Comment:
hmm, I think this could be too easily confused for an option that allows updates in the non-ROB case.

perhaps we should name it "reorderBufferAllowUpdate" or "allowROBUpdate" ?

Reply:
true, done in 96e11df